docs/install.md

```diff
@@ -243,38 +246,105 @@ attribute, and include the appropriate files.
 
 | Dependency | Headers provided | Notes |
 |------------|------------------|-------|
-| `@au//au` | `"au/au.hh"`<br>`"au/units.*.hh"` | Core library functionality.  See [all available units](https://github.com/aurora-opensource/au/tree/main/au/units) |
+| `@au//au` | `"au/au.hh"`<br>`"au/units/*.hh"` | Core library functionality.  See [all available units](https://github.com/aurora-opensource/au/tree/main/au/units) |
 | `@au//au:io` | `"au/io.hh"` | `operator<<` support |
-| `@au//au:testing` | `"au/testing.hh"` | Utilities for testing<br>_Note:_ `testonly = True` |
+| `@au//au:testing` | `"au/testing.hh"` | Utilities for writing googletest tests<br>_Note:_ `testonly = True` |
 
 #### CMake
 
-CMake support is still experimental and in-progress.  We are building it up starting from "root"
-targets (that is, targets without dependencies), and expanding support to the rest of the library.
+There are two ways to include the Au library in your CMake project.
 
-To build the library using this experimental CMake support, follow these steps:
+1. **(Recommended)** Use the `FetchContent` module to download the library directly from GitHub.
 
-```sh
-# CMake is a "meta build system", not a build system.
-# This first command generates the actual build files.
-cmake -S . -B cmake/build -DCMAKE_VERIFY_INTERFACE_HEADER_SETS=TRUE
+2. Install the library to the system, and use `find_package`.
 
-# This command builds Au, checks include paths, and runs unit tests.
-cmake \
-  --build cmake/build \
-  --target \
-    all \
-    all_verify_interface_header_sets \
-    test
-```
+The reason we recommend `FetchContent` is because it makes it easier for each project to get the
```

Comment:
This wording is a little clunky. It might be clearer to write something like
```
We recommend `FetchContent` because it gets the exact version and installs au for the project.
```

Reply:
I tweaked it a little bit based on your recommendation.